barcodes_generator_abstract/tests/test_barcodes_generator_abstract.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2021 Tecnativa - Carlos Roca
+# Copyright 2023-Today GRAP (http://www.grap.coop)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo_test_helper import FakeModelLoader
+
+from odoo.exceptions import UserError
+from odoo.tests import TransactionCase
+
+
+class TestBarcodesGeneratorAbstract(TransactionCase, FakeModelLoader):
```

Comment:
```suggestion
class TestBarcodesGeneratorAbstract(BaseCommon, FakeModelLoader):
```



Use BaseCommon as base test class to reduce overhead from tracking and boost the test suite.

Reply:
for reference: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0#tasks-to-do-in-the-migration